java/spacewalk-java.changes

```diff
@@ -6,7 +6,6 @@ Tue Feb 21 14:12:10 CET 2023 - jgonzalez@suse.com
   * set uptime at package profile update
   * Install the reboot info beacon using a conf file instead of using pillars
   * Handle taskomatic failures during action creation
-  * Reduce taskomatic memory consumption while processing Ubuntu Erratas
```

Comment:
This was already tagged, so it must not be removed.

Instead of removing the line here, add a new one on top:
```
- Rollback changes to reduce taskomatic memory consumption while processing
  Ubuntu Erratas

```